token-lending/program/src/processor.rs

```diff
@@ -1344,10 +1344,11 @@ fn process_borrow_obligation_liquidity(
         msg!("Borrow reserve is stale and must be refreshed in the current slot");
         return Err(LendingError::ReserveStale.into());
     }
-    if Decimal::from(liquidity_amount)
```

Comment:
Do we want to do something similar for deposit limits?

Reply:
naw cause they dont have a max for deposits